examples/fft_conv.cu

```diff
@@ -104,17 +112,39 @@ int main([[maybe_unused]] int argc, [[maybe_unused]] char **argv)
 
   // Prefetch the data we just created
   sig_time.PrefetchDevice(0);
-  filt_time.PrefetchDevice(0);
+  filt_time.PrefetchDevice(0);  
+
 
   // Perform the FFT in-place on both signal and filter
-  fft(sig_freq, sig_time);
-  fft(filt_freq, filt_time);
+  for (int i = 0; i < iterations; i++) {
+    if (i == 1) {
+      cudaEventRecord(start, stream);
```

Comment:
Is this meant to be a benchmark?  if so maybe put it in bench instead and just leave the example as an example without iterations/timing?